json-schemas/dpi-acknowledge.json

```diff
@@ -1,13 +1,13 @@
 {
   "$id": "https://schemas.ruter.no/adt/ota/api/v2.1/proprietary-extensions/dpi-acknowledge.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "DPIAcknowledge",
+  "title": "DPI Acknowledge",
   "type": "object",
   "required": [
     "eventTimestamp",
     "traceId",
-    "type",
-    "payload"
+    "clientId",
```

Comment:
Skal `clientId` være required? Hva skal settes i dette feltet for utvendig skilt og lyd?

Reply:
Tenker det er en fordel å få clientId uansett, slik at det er lettere å tracke hvilket utstyr som responderer for både operatør og oss.